src/main/java/com/google/api/codegen/ruby/RubyGapicContext.java

```diff
@@ -382,6 +384,29 @@ private SurfaceTransformerContext getSurfaceTransformerContextFromService(Interf
         new RubyFeatureConfig());
   }
 
+  public LongRunningConfig getLongrunningConfig(Method method, Interface service) {
+    return getApiConfig()
+        .getInterfaceConfig(service)
+        .getMethodConfig(method)
+        .getLongRunningConfig();
+  }
+
+  public boolean isLongrunning(Method method, Interface service) {
+    return getApiConfig()
+        .getInterfaceConfig(service)
+        .getMethodConfig(method)
+        .isLongRunningOperation();
+  }
+
+  public boolean hasLongrunningMethod(Interface service) {
+    for (Method method : getSupportedMethodsV2(service)) {
+      if (getLongrunningConfig(method, service) != null) {
```

Comment:
I think calling 'isLongrunning(method, service)' is  a bit better here since it's more self-explaining and isLongRunningOperation is checking 'getLongRunningConfig() != null'

Also please use 'LongRunning' instead of 'Longrunning' in order to be consistent.